dts/arm/st/h7/stm32h753Xi.dtsi

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020 Moonkwun Jung <mkainyh@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/h7/stm32h753.dtsi>
+
+/ {
+	soc {
+		flash-controller@52002000 {
+			flash0: flash@8000000 {
+				reg = <0x08000000 DT_SIZE_K(1024)>;
```

Comment:
Flash size for 'i' variant is 2M

Reply:
It is my mistake. I will revise it to reflect your opinion.

Reply:
thanks